### ISSUE TITLE
New marketplace experience

### ISSUE THREAD

**Issue by andersonjeccel** [mautic/mautic#13143]
### Description
The project aims to create a fresh and improved marketplace experience with enhanced features, design, and usability.
### Impact
The new marketplace experience will likely result in increased user satisfaction, easier navigation, and potentially higher adoption rates, contributing to a more vibrant and successful marketplace ecosystem.

**Comment by RCheesley** [comment]
Still relevant

**Comment by andersonjeccel** [comment]
@RCheesley I'm about to remove the marketplace from my planning. Does someone want me to tackle a redesign there?

**Comment by RCheesley** [comment]
Yes, I think we will make it a Summer of Code project next year. We will need to write up a brief though, so it would be hugely helpful if you'd be up for being involved with that?
